### DISCUSSION TITLE
Only remote/local DRAM access is displayed in pcm-numa. What should I do if I want to observe the access status of pmm?

### DISCUSSION THREAD

**Discussion by Yxiaoge** [opcm/pcm#260] [Q&A] [answered]
Only remote/local DRAM access is displayed in pcm-numa. What should I do if I want to observe the access status of pmm?
Thanks, Looking forward to your reply.

**Comment by opcm** [comment, marked as answer]
you can use pmu-query.py to query MEM_LOAD_RETIRED.LOCAL_PMM and MEM_LOAD_L3_MISS_RETIRED.REMOTE_PMM event encodings and then use them in pcm-core utility. This should give at least the distribution of local/remote loads for PMM.

**Comment by opcm** [comment]
Here is another example of collecting arbitrary core events with pcm: https://github.com/opcm/pcm/discussions/255#discussioncomment-193599
